packages/storybook/.storybook/preview-head.html

```diff
@@ -112,4 +112,8 @@
         background: #faf9f8;
         color: #605e5c;
     }
+
```

Comment:
Does this affect all stories?

Reply:
It affect all storybook but I haven't found any usage of figure tag in our storybook (when ran the storybook and checked Elements tab)